azuread/resource_application.go

```diff
@@ -121,6 +170,7 @@ func resourceApplicationCreate(d *schema.ResourceData, meta interface{}) error {
 	name := d.Get("name").(string)
 
 	properties := graphrbac.ApplicationCreateParameters{
+		AdditionalProperties:    make(map[string]interface{}),
```

Comment:
What is the reason for this here if we aren't setting anything? will the API not return it otherwise?

Reply:
This is a fragment left from #78. Will remove this here.